interfaces/HO-Portal/src/app/program/program-payout/program-payout.component.html

```diff
@@ -35,6 +38,16 @@ <h3>{{ 'page.program.program-payout.make-export'|translate }}</h3>
       [paymentInstallment]="exportInstallmentId"
       [disabled]="!exportInstallmentAvailable"
     ></app-export-list>
+    <ng-container>
+      <app-export-list *ngIf="programId"
+      [programId]="programId"
+      [exportType]="enumExportType.payment"
+      [minInstallment]=10
```

Comment:
Is this number based on something?
I think hard-coding `[minInstallement]="0"` could be 'safe enough', but the `maxInstallment` should be related to "the most recent succesfull installment"(so `lastInstallmentId` or similar) or "the maximun installment possible for this program"(so `program.distributionDuration`)